clueless/models.py

```diff
@@ -33,8 +33,8 @@ class Space(models.Model):
     """
     posX = models.IntegerField()
     posY = models.IntegerField()
-    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True)
-    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True)
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True, null=True)
+    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True, null=True)
     spaceCollector = models.ForeignKey(SpaceCollection)
```

Comment:
So I had to look up what null = true does but according to django documentation doing this is potentially bad and unnecessary if we have blank=true also? I might be misunderstanding why you added this but I don't know if we need it

https://docs.djangoproject.com/en/1.10/ref/models/fields/